germanium/client.py

```diff
@@ -1,211 +0,0 @@
-from __future__ import unicode_literals
-
-import logging
-
-from django.test.client import Client
-from django.test.testcases import LiveServerTestCase
-from django.test.client import RequestFactory
-
-from germanium.auth import AuthTestCaseMixin
-from germanium import config
-from germanium.asserts import AssertMixin
-from germanium.testcase import GermaniumTestCase
-
-
-class ClientTestCase(GermaniumTestCase, AuthTestCaseMixin, LiveServerTestCase, AssertMixin):
```

Comment:
tady je jedna zásadní chyba dědí to z LiveServerTestCase alejak je psáno v dokumentaci LiveServerTestCase spouští server na určitém portu nicméně pokd se použivá Client tak se volá přímo django funkce a vůbec se spuštěná aplikace na portu nepoužívá. Cílem LiveServerTestCase je napříkad testování seleniem Tohle mohlo způsobovat slušné zpomalení testů